pandas/_typing.py

```diff
@@ -23,6 +23,7 @@
     from pandas.core.indexes.base import Index  # noqa: F401
     from pandas.core.series import Series  # noqa: F401
     from pandas.core.generic import NDFrame  # noqa: F401
+    from pandas.io.formats.style import Styler  # noqa: F401
```

Comment:
Just put this in a `if TYPE_CHECKING` block in pandas.core.frame instead of here